src/peripheral/mod.rs

```diff
@@ -130,6 +131,9 @@ pub struct Peripherals {
     /// Nested Vector Interrupt Controller
     pub NVIC: NVIC,
 
+    /// Security Attribution Unit
```

Comment:
This is a breaking change. Adding a field to a structure that only has public fields is a breaking change.

"Why?" Because of destructuring syntax:

``` rust
fn foo(a: A) {
    let A { b, c } = a;
}

struct A {
    pub b: u8,
    pub c: u16,
    // pub d: u32, // adding this breaks the above `let` statement
}
```